plugins/modules/foreman_role.py

```diff
@@ -54,6 +54,16 @@
     description: Filters with permissions for this role
     required: false
     type: list
+    elements: dict
+    suboptions:
```

Comment:
the API also supports an [`override`](https://theforeman.org/api/1.23/apidoc/v2/filters/create.html) param, should this be added too (or in an own PR?)

Reply:
As this one is more like a bugfix PR, i'd put that in a separate feature PR.